app/models/user.js

```diff
@@ -0,0 +1,9 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  first_name: DS.attr('string'),
+  last_name: DS.attr('string'),
+  email: DS.attr('string'),
+  team: DS.attr('string'),
+  joiningDate: DS.attr('date')
```

Comment:
Use camelCase in the model attribute names.

Reply:
I have fixed this change.